src/_P035_IRTX.ino

```diff
@@ -353,12 +337,13 @@ boolean Plugin_035(byte function, struct EventStruct *event, String& string)
 }
 
 boolean addErrorTrue() {
-    addLog(LOG_LEVEL_ERROR, F("RAW2: Invalid encoding!"));
+    addLog(LOG_LEVEL_ERROR, String(F("RAW2: Invalid encoding!")));
```

Comment:
Why do you need this extra String wrapper?

Reply:
in this cases like those, if you do not wrap it, it wont compile.
`addLog(LOG_LEVEL_ERROR, String(F("RAW2: Invalid encoding!")) + "ABCD");`
In the example you wrote about the `String `is not nessesary